src/pages/docs/getting-started/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Getting Started"
+description: "Metal.js is a javascript library for building UI components in a solid and flexible way."
+layout: "guide"
+icon: "flash"
+weight: 1
+---
+
+###### {$page.description}
+
+<article id="1">
+
+## Introduction
+
+There are many different ways to build components using Metal.js. You can use the built-in integrations with [Soy](http://developers.google.com/closure/templates/) or [JSX templates](https://facebook.github.io/jsx/), or even build your own template abstraction on top of Metal.js and use it instead.
+
+You can build the ES6 code using babel, webpack or any other package bundler. You can test using karma, jasmine or any other test framework, and even use any directory structure for your project. To sum it up, you can customize everything to your own needs. The world is your oyster.
```

Comment:
https://github.com/metal/metaljs.com/search?utf8=%E2%9C%93&q=build+the+ES6+code+using+babel&type=